demofile-parsing/parse_demo.js

```diff
@@ -1,24 +1,119 @@
 const fs = require("fs");
 const demofile = require("demofile");
 const path = require("path");
+const Round = require('./Round.js');
 
-fs.readFile(path.resolve("/Users/tylerlam/Downloads/test.dem"), (err, buffer) => {
+const DEBUG = false;
```

Comment:
@tjlam Is this a JS y thing that will flip the switch and become true if something hits the fan?

Reply:
I think you have to manually flip it to true for testing purposes. No where in the code it returns to true.